docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -408,13 +405,10 @@ desired={<Eight viewBox="140 40 680 500" />}
 
 <div>
 
-#### Current state
-
-You have a standalone region with a working Camunda 8 setup, including Zeebe, Operate, Tasklist, and Elasticsearch.
-
-#### Desired state
-
-You want to restore the dual-region functionality and deploy Camunda 8, consisting of Zeebe and Elasticsearch, to the newly restored region. Operate and Tasklist need to stay disabled to prevent interference with the database backup and restore.
+| **Details**              | **Current State**                                                                                                   | **Desired State**                                                                                                 |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| **Camunda 8 Setup**      | A standalone region with a fully functional Camunda 8 setup, including Zeebe, Operate, Tasklist, and Elasticsearch. | Restore dual-region functionality by deploying Camunda 8 (Zeebe and Elasticsearch) to the newly restored region.  |
+| **Operate and Tasklist** | Operate and Tasklist are operational in the standalone region.                                                      | Operate and Tasklist should remain disabled to avoid interference during the database backup and restore process. |
```

Comment:
the rework on the wording is not ideal
> Operate and Tasklist should remain disabled to avoid interference during the database backup and restore process.

I'd stay with
> ... need to stay disabled to prevent ...

should has an optional flair to it and makes it sounds like it's okay if you have it enabled and avoid is also not ideal vs prevent. We're not avoiding this, we're trying to prevent it from happening in the first place.